lib_nbgl/src/nbgl_use_case.c

```diff
@@ -2401,6 +2399,23 @@ void nbgl_useCaseSettings(const char                *title,
     displaySettingsPage(initPage, true);
 }
 
+/**
+ * @deprecated
+ * See #nbgl_useCaseNavigableContent
+ */
+void nbgl_useCaseSettings(const char                *title,
```

Comment:
here we could mention `useCaseHomeAndSettings`

Reply:
done